src/promptflow-devkit/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -360,7 +361,7 @@ def _visualize(self, runs: List[Run], html_path: Optional[str] = None) -> None:
                 metrics=local_storage.load_metrics(parse_const_as_str=True),
                 dag=local_storage.load_dag_as_string(),
                 flow_tools_json=local_storage.load_flow_tools_json(),
-                mode="eager" if local_storage.eager_mode else "",
+                mode=EAGER.lower() if local_storage.eager_mode else "",
```

Comment:
this constant name is a little confusing. What about put it under a class like:
```python
class RunMode:
  EAGER = "eager"
```

Reply:
Yes, I will modify it.